homeassistant/components/dnsip/sensor.py

```diff
@@ -92,7 +105,11 @@ async def async_update(self) -> None:
             response = None
 
         if response:
-            self._attr_native_value = response[0].host
+            sorted_ips = sort_ips(
+                [res.host for res in response], querytype=self.querytype
+            )
+            self._attr_native_value = sorted_ips[0]
+            self._attr_extra_state_attributes["IPs"] = sorted_ips
```

Comment:
Sorry for coming back late on this one but could we change the attribute to `ip_addresses` and then also add a translation in `strings.json` 👍